package.nls.json

```diff
@@ -2,11 +2,13 @@
     "azure-account.cloudShellBash": "Azure Cloud Shell (Bash)",
     "azure-account.cloudShellPowerShell": "Azure Cloud Shell (PowerShell)",
     "azure-account.commands.azure": "Azure",
+    "azure-account.commands.azureAccount": "Azure Account",
```

Comment:
Maybe `Azure Account Extension`? I'm worried it will be confused for a way for anybody with essentially any problems with their Azure account to report issues. @fiveisprime want to provide some PM input too?

Reply:
I do get the concern. I think worst case we can always direct people who file issues related to general Azure or Azure accounts to the right place

Reply:
Yeah, I think even if we added Azure Account Extension, we'll still issues filed against us for general Azure Account problems because I don't think most users will be able to distinguish when it's something that our extension is/is not doing vs. their Azure Account is just busted.

And even if it's a problem with their Azure Account, there may be steps that we can take in the extension to resolve/guide them as well.